docs/content/manual/pre-release/cluster-restore/restore-to-a-new-cluster.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Restore to a new cluster
+---
+
+#### Back up the old cluster
+1. Deploy the 1st cluster then install Longhorn system and Velero.
+2. Deploy some workloads using Longhorn volumes then write some data:
+   1. A simple pod using multiple volumes. And some volumes are using backing images.
+   2. A StatefulSet.
+   3. A Deployment with a RWX volume.
+3. Config some recurring policies for the volumes.
+4. Create backups for all volumes.
+5. Create a cluster backup via Velero.
+    ```bash
+    velero backup create lh-cluster --exclude-resources persistentvolumes,persistentvolumeclaims,backuptargets.longhorn.io,backupvolumes.longhorn.io,backups.longhorn.io,nodes.longhorn.io,volumes.longhorn.io,engines.longhorn.io,replicas.longhorn.io,backingimagedatasources.longhorn.io,backingimagemanagers.longhorn.io,backingimages.longhorn.io,sharemanagers.longhorn.io,instancemanagers.longhorn.io,engineimages.longhorn.io
+    ```
+   
+#### Restore to a new cluster
+1. Deploy the 2nd cluster then install Velero only. You can try with different cluster config (more nodes or disks) here.
+2. Restore the cluster backup. e.g.,
+    ```bash
+    velero restore create --from-backup lh-cluster
+    ```
+3. Removing all old instance manager pods from namespace `longhorn-system`.
```

Comment:
Could you explain why we need removing all old instance manager pods here? I guess you would like to clean up old IM pods recreated by Velero but instead just delete them first then let longhorn managers recreate them.

For managed pods, is it possible any other cases similar to IM pods? like backing image?

Reply:
Yes. We need to clean up the instance manager pods created by Velero. There is no corresponding InstanceManager CR for these old pods.
And thank you for reminding me. It's better to remove backing image manager pods as well.